ballots/tests.py

```diff
@@ -0,0 +1,95 @@
+import datetime
+from django.contrib.auth.models import User
+from django.test import TestCase, RequestFactory
+from django.urls import reverse
+from django.utils import timezone
+
+from .models import Ballot
+from users.models import Profile
+from .forms import AddBallotForm
+
+class BallotModelTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        User.objects.create(first_name='John', last_name='Smith', username = 'testuser', password='12345')
+        profile = User.profile
+        profile.ssn = "555-55-5555"
+        profile.district = "BaltimoreCounty"
+        profile.middle_name = "Jack"
```

Comment:
This really should have been something caught in the user profile PR, but I didn't know well enough to say anything about it until now:

You need to store the value of this user within your test class so that you can refer back to it later in your classes' tests.
```python
""" line 14 """ self.user = User.objects.create(...)
```
There is actually **no guarantee that the user will be created with a specific ID**, so you can not grab your user from the DB in later tests using static IDs like so:
```python
# Should not be doing this
user = User.objects.get(id=1)
assertEqual(test_ballot.district, user.profile.district)

# Should be this instead
assertEqual(test_ballot.district, self.user.profile.district)

# OR, altnernatively but not preferably
user = User.objects.get(id=self.user.id) 
assertEqual(test_ballot.district, user.profile.district)
```
If you run `python manage.py test`, the users app tests all fail under our current test code, because the first test user from our ballot app tests is created with an ID of 1, but the second user is created with an ID of 2. Please update your ballot tests to comply! I'll make a PR for fixing the user tests.

Reply:
I'm going to use the worse method for now because the setupdata function doesn't have access to the self variable. I'll work out how to make that happen. I think something I was looking into called test_factory (iirc) might be the solution.